docs/articles_en/documentation/openvino-ir-format/operation-sets/available-opsets/opset14.rst

```diff
@@ -174,7 +174,7 @@ Table of Contents
 * :doc:`Round <../operation-specs/arithmetic/round-5>`
 * :doc:`ScaledDotProductAttention <../operation-specs/sequence/scaled-dot-product-attention>`
 * :doc:`ScatterElementsUpdate <../operation-specs/movement/scatter-elements-update-12>`
-* :doc:`ScatterNDUpdate <../operation-specs/movement/scatter-nd-update-3>`
+* :doc:`ScatterNDUpdate <../operation-specs/movement/scatter-nd-update-14>`
```

Comment:
Let's not overwrite ScatterNDUpdate in Opset14 and add the spec to the Opset15 (init the Opset15 doc file with the minimum set of the ops if needed)